app/models/manageiq/providers/amazon/container_manager.rb

```diff
@@ -0,0 +1,14 @@
+class ManageIQ::Providers::Amazon::ContainerManager < ManageIQ::Providers::Kubernetes::ContainerManager
+  require_nested :EventCatcher
+  require_nested :EventParser
+  require_nested :Refresher
+  require_nested :RefreshWorker
+
+  def self.ems_type
+    @ems_type ||= "eks".freeze
```

Comment:
Moving forward, if we could have sensible defaults for the various settings keys, then this can act as the anchor key, and all of the rest can front from it.  In fact, maybe it's better to have this at the engine.rb

Reply:
Well the engine is for the whole plugin where this is per manager

Reply:
Ah yeah true...I still would love to specify less overrides in the child objects though.  So yeah, perhaps the ems_type stays here, but still becomes an anchor.

Reply:
Oh yeah :100: %